frontend/src/components/assessments/assessment-creation/QuestionSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
can you refactor `StudentAssessment.tsx` to use this component?

Reply:
yes maam